### PR TITLE
Effect Deadds OK

### PR DESCRIPTION
**Pull request by hensldm**

### CODE REVIEW

File: src/overlays/effects/ovl_Effect_Ss_Dead_Ds/z_eff_ss_dead_ds.c [hunk 1]
```diff
@@ -1,27 +1,124 @@
...
+    }
+
+    Matrix_Translate(this->pos.x, this->pos.y, this->pos.z, MTXMODE_NEW);
+    Matrix_RotateZYX(this->rRoll, this->rPitch, this->rYaw, MTXMODE_APPLY);
+    Matrix_RotateXFApply(1.57f);
+    Matrix_Scale(scale, scale, scale, MTXMODE_APPLY);
+    gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+    gDPSetCombineLERP(POLY_XLU_DISP++, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE, 0, 0, 0, 0, PRIMITIVE, TEXEL0, 0,
+                      PRIMITIVE, 0);
+    gSPDisplayList(POLY_XLU_DISP++, gLensFlareCircleDL);
```
Comment (EllipticEllipsis) [inline src/overlays/effects/ovl_Effect_Ss_Dead_Ds/z_eff_ss_dead_ds.c:106]: Hmm, this name may need a rethink, although I'm guessing it's used elsewhere.

Comment (hensldm) [reply src/overlays/effects/ovl_Effect_Ss_Dead_Ds/z_eff_ss_dead_ds.c:106]: Based on OoT used in Kankyo.

**Review by EllipticEllipsis — APPROVED** [review]

File: src/overlays/effects/ovl_Effect_Ss_Dead_Ds/z_eff_ss_dead_ds.c (hunk 1 above)
Comment (engineer124) [reply src/overlays/effects/ovl_Effect_Ss_Dead_Ds/z_eff_ss_dead_ds.c:106]: Seems like they really liked reusing the lens textures/display lists for other things in MM. I know we currently switched `gLensOfTruthMaskTex` -> `gCircleTex` (though that name is probably far to generic tbh, might also need a rethink), but yeah, not sure what to do with these. Might not be worth rethinking until we have all use-cases (near 100% decomp) so we can get a bigger picture.

Comment (hensldm) [reply src/overlays/effects/ovl_Effect_Ss_Dead_Ds/z_eff_ss_dead_ds.c:106]: Pretty sure that is something different. This is part of gameplay keep and looks to be used in just `DeadDd`, `DeadDs`, and `kankyo`.

**Review by engineer124 — APPROVED** [review]

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**